schemas/2017-04-01/Microsoft.Relay.json

```diff
@@ -125,7 +125,7 @@
         "type": {
           "type": "string",
           "enum": [
-            "Microsoft.Relay/namespaces/hybridConnection"
+            "Microsoft.Relay/namespaces/hybridConnections"
```

Comment:
>hybridConnections [](start = 40, length = 17)

Please update schemas/2015-01-01/deploymentTemplate.json and schemas/2014-04-01-preview/deploymentTemplate.json for these changes.

Reply:
Have updated the DeplymentTemplate.json